hash.c

```diff
@@ -1837,10 +1837,7 @@ rb_hash_initialize(int argc, VALUE *argv, VALUE hash)
  *    Hash[:foo, 0, :bar, 1] # => {:foo=>0, :bar=>1}
  *
  *  When argument <tt>hash_convertible_object</tt> is given,
```

Comment:
We may want to change the argument name to just `hash` (could be done in a later commit, but might be better done in this one).

Reply:
Fixed.